tools/store/backend-matrix.sh

```
@@ -0,0 +1,88 @@
+pass=0
+fail=0
+
+# Every valid selection must compile.
+for peer in "" \
+    "--no-default-features --features kyotocabinet" \
+    "--no-default-features --features redb" \
+    "--no-default-features --features lmdb" \
+    "--no-default-features --features tkrzw"; do
+    label=${peer:-default (KC)}
+    printf '── valid: %s\n' "$label"
+    if cargo check --locked -p oxpinyin-store $peer >/tmp/backend-matrix-$$.log 2>&1; then
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)